### PR TITLE
feat: [SRM-10238]: Added created and modified dates in SLO details page

### PR DESCRIPTION
**Pull request by shashidhargg**
##### Summary:

- Keep the time filter for switching SLO and Error budget tabs.
- Show the current day selected in the custom calendar.
- Add created and last modified date on the Header.

##### Jira Links:

https://harness.atlassian.net/browse/SRM-10238

##### Screenshots:

Screenshot 2022-04-27 at 3 30 02 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/85-cv/pages/slos/CVSLODetailsPage/views/HeaderToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <Container height={12} width={200} className={Classes.SKELETON} />
+        <Container height={12} width={200} className={Classes.SKELETON} />
+      </Layout.Vertical>
+    )
+  }
+
+  return (
+    <Layout.Vertical height="100%" flex={{ justifyContent: 'flex-end', alignItems: 'end' }}>
+      <Text font={{ variation: FontVariation.TINY_SEMI }} color={Color.GREY_500}>
+        {getString('cv.lastModifiedOn')} {moment(new Date(createdAt)).format('D MMM YYYY h:mm:ss A')}
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/CVSLODetailsPage/views/HeaderToolbar.tsx:30]: shall we move this to a variable for better readability?

File: src/modules/85-cv/pages/slos/CVSLODetailsPage/views/HeaderToolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      </Text>
+      <Text font={{ variation: FontVariation.TINY_SEMI }} color={Color.GREY_500}>
+        {getString('cv.createdOn')} {moment(new Date(lastModifiedAt)).format('D MMM YYYY h:mm:ss A')}
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/CVSLODetailsPage/views/HeaderToolbar.tsx:33]: same here

File: src/modules/85-cv/pages/slos/SLOCard/TimeRangeFilter.tsx [hunk 3]
```diff
@@ -93,6 +93,7 @@ const TimeRangeFilter: React.FC<SLOTargetChartWithChangeTimelineProps> = ({
         }
         dateRangePickerProps={{
           shortcuts: false,
+          ...(customTimeFilter ? { defaultValue: [new Date(startTime), new Date(endTime)] } : {}),
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/SLOCard/TimeRangeFilter.tsx:96]: please wrap this in a function

**Review by praneshg239 — APPROVED** [review]

**Comment by shashidhargg** [comment]
trigger cypress

**Comment by shashidhargg** [comment]
trigger rebase

**Pull request merged**